src/components/panels/errordisplays.tsx

```diff
@@ -121,7 +121,7 @@ export function DeveloperWarningSourceDisplay({
 
   return (
     <div className={classes.validationErrorContext}>
-      In {/* @ts-expect-error RRv5 types are jank */}
+      In
       <Link className={classes.link} to={DataStorePaths.Schema()}>
```

Comment:
Links work the same way and they don't make typescript angry anymore :raised_hands: